airbyte-persistence/job-persistence/src/test/java/io/airbyte/persistence/job/DefaultJobPersistenceTest.java

```diff
@@ -468,6 +469,57 @@ void testGetStatsNoResult() throws IOException {
 
     }
 
+    @Test
```

Comment:
An idea to make this test slightly easier to read/debug - you could introduce some constants like 
```
JOB_1_ATTEMPT_1_WRITE_1_EMITTED_RECORDS = 500L;
JOB_1_ATTEMPT_1_WRITE_2_EMITTED_RECORDS = 1000L;
...
```
So that you can reference these in the `exp` map to make it very obvious what the expected values are. Might be a huge pain with so many constants though so I'll leave it up to you!

Reply:
I was implementing it halfway when I gave up.. there are too many variables such that the naming + horizontal spacing becomes huge 😢  I'm going to leave it as it now if you don't mind!

Reply:
haha yeah totally reasonable, figured it would be really hairy